lib/core/src/lib/auth/guard/auth-guard-sso-role.service.ts

```diff
@@ -16,14 +16,14 @@
  */
 
 import { Injectable } from '@angular/core';
-import { ActivatedRouteSnapshot, CanActivate, Router } from '@angular/router';
+import { ActivatedRouteSnapshot, Router } from '@angular/router';
 import { MatDialog } from '@angular/material/dialog';
 import { UserAccessService } from '../services/user-access.service';
 
 @Injectable({
     providedIn: 'root'
 })
-export class AuthGuardSsoRoleService implements CanActivate {
+export class AuthGuardSsoRoleService  {
```

Comment:
is this something the angular upgraade does? we don't need CanActivate anymore?

Reply:
it does seem so i haven't changed anything in the code in this stage, just checking the upgrade